.gitignore

```diff
@@ -1,4 +1,6 @@
 .generated
+.idea
```

Comment:
Not sure what your standards are, but I thought it'd be fine to add an IDE specific (IntelliJ in this instance) folder to `.gitignore`.